android/gradle.properties

```diff
@@ -6,11 +6,11 @@ android.enableJetifier=true
 # Defining them here makes them available both in
 # settings.gradle and in the build.gradle files.
 
-agpVersion=8.1.4
+agpVersion=8.5.0
```

Comment:
Is there a reason to go for 8.5.0 instead of 8.5.2? It looks like that's the latest.

(My usual approach would be to go for the latest patch release within a given minor release, even when holding back at an older minor or major release for whatever reason.)

Reply:
> Is there a reason to go for 8.5.0 instead of 8.5.2?

Not really, hadn't realized there was a newer one, I just took that version from release notes.
Updated now to 8.5.2.